inst/templates/ad_adlb.R

```diff
@@ -228,6 +222,74 @@ adlb <- adlb %>%
   derive_var_pchg()
 
 
+## Calculate lab grading ----
+
+# Assign ATOXDSCL and ATOXDSCH to hold lab grading terms
+# ATOXDSCL and ATOXDSCH hold terms defined by NCI-CTCAEv4.
+grade_lookup <- tibble::tribble(
+  ~PARAMCD, ~ATOXDSCL,                      ~ATOXDSCH,
+  "ALB",     "Hypoalbuminemia",             NA_character_,
+  "ALKPH",   NA_character_,                 "Alkaline phosphatase increased",
+  "ALT",     NA_character_,                 "Alanine aminotransferase increased",
+  "AST",     NA_character_,                 "Aspartate aminotransferase increased",
+  "BILI",    NA_character_,                 "Blood bilirubin increased",
+  "CA",      "Hypocalcemia",                "Hypercalcemia",
+  "CHOLES",  NA_character_,                 "Cholesterol high",
+  "CK",      NA_character_,                 "CPK increased",
+  "CREAT",   NA_character_,                 "Creatinine increased",
+  "GGT",     NA_character_,                 "GGT increased",
+  "GLUC",    "Hypoglycemia",                "Hyperglycemia",
+  "HGB",     "Anemia",                      "Hemoglobin increased",
+  "POTAS",   "Hypokalemia",                 "Hyperkalemia",
+  "LYMPH",   "CD4 lymphocytes decreased",   NA_character_,
+  "PHOS",    "Hypophosphatemia",            NA_character_,
+  "PLAT",    "Platelet count decreased",    NA_character_,
+  "SODIUM",  "Hyponatremia",                "Hypernatremia",
+  "WBC",     "White blood cell decreased",  "Leukocytosis",
+)
+
+# add ATOXDSCL and ATOXDSCH
+adlb <- adlb %>%
+  derive_vars_merged(
+    dataset_add = grade_lookup,
+    by_vars = vars(PARAMCD),
+  ) %>%
+  # derive toxicity grade for low values ATOXGRL
+  derive_var_atoxgr_dir(
+    new_var = ATOXGRL,
+    tox_description_var = ATOXDSCL,
+    criteria_direction = "L",
+    get_unit_expr = extract_unit(PARAM)
+  ) %>%
+  # derive toxicity grade for low values ATOXGRH
```

Comment:
```suggestion
# Add ATOXDSCL and ATOXDSCH
adlb <- adlb %>%
  derive_vars_merged(
    dataset_add = grade_lookup,
    by_vars = vars(PARAMCD),
  ) %>%
  # Derive toxicity grade for low values ATOXGRL
  derive_var_atoxgr_dir(
    new_var = ATOXGRL,
    tox_description_var = ATOXDSCL,
    criteria_direction = "L",
    get_unit_expr = extract_unit(PARAM)
  ) %>%
  # Derive toxicity grade for low values ATOXGRH
```

CAn you start the comments with proper case please

Reply:
Done, thanks Ben.